bento/src/main/java/com/yelp/android/bento/components/ListComponent.java

```diff
@@ -253,6 +255,21 @@ public int getSpanSize(int position) {
         };
     }
 
+    @Override
+    public void onItemsMoved(int oldIndex, int newIndex) {
+        super.onItemsMoved(oldIndex, newIndex);
+
+        mData.add(newIndex, mData.remove(oldIndex));
+
+        if (mOnItemMovedCallback != null) {
+            mOnItemMovedCallback.onItemMoved(oldIndex, newIndex, mData);
```

Comment:
@dwaxemberg Is it ok to be passing back a raw `mData`, or should I be passing a copy of the list?

Reply:
Ah, this was needed at one time, but not since we added the absolute position to the ViewHolders. I'll remove it.